blog/_posts/2016-09-20-sip-meeting-september-results.md

```diff
@@ -0,0 +1,81 @@
+-   [SIP-26: Unsigned Integer Data Types](http://docs.scala-lang.org/sips/rejected/unsigned-integers.html).
+    Proposed by Sébastien Doeraene and Denys Shabalyn, unsigned integers have
+    been greatly discussed [here](https://github.com/scala/slip/pull/30) and
+    [here](https://github.com/scala/scala.github.com/pull/548). Three months
+    ago, when the proposal was numbered, Sébastien Doeraene convinced the
+    Committee to try to improve the performance hit in the provided
+    implementation, for which he asked for two months. Unfortunately, after some
+    work during two months, a 6% performance hit remained. As a result,
+    Sébastien proposed to reject the proposal, whose motion was unanimously
+    approved by the Committee. For more information, have a look at his full
+    report [here](https://github.com/scala/scala.github.com/pull/548).
+-   [SIP-27: Trailing Commas](http://docs.scala-lang.org/sips/pending/trailing-commas.html).
+		The trailing commas proposal was acknowledged and numbered by the Committee
+		three months ago. This time, Dale Wijnand, the author, was invited as a
+		guest and took part in the discussion. After some insightful discussions by the community
+		[here](https://github.com/scala/scala.github.com/pull/533) and
+		[there](https://gitter.im/scala/slip), and much more debate in our
+		meetings, the Committee has accepted the proposal. As releasing is not part
+		of the new process, the Committee hands it over to the Compiler team which,
+		based on the compiler compatibility guarantees, will pick the closest Scala
+		release in which trailing commas can be shipped.
+
+Do you want to know more about our latest meeting? Check our [minutes](http://docs.scala-lang.org/sips/minutes/sip-20th-september-minutes.html).
+
+Overall, we're also excited to announce the first accepted proposal since
+our major rethink of the SIP process. Three months ago, The Scala Center
+released the new Scala Improvement Process with two main features in mind:
+shortening the duration of proposal reviews and making it easier for authors to
+contribute and get involved. We're particularly happy to see that such efforts
+have been useful for the community and the future of the language. We hope to
+see more community involvement in the upcoming months, and we encourage any
+Scala developer with interesting ideas to
+[participate](http://docs.scala-lang.org/sips/sip-submission.html)!
+
+Next month, we’ll discuss four more proposals, including
+[SIP-20: Improved lazy val initialization](http://docs.scala-lang.org/sips/pending/improved-lazy-val-initialization.html) and
+[SIP-24: Repeated by-name parameters](http://docs.scala-lang.org/sips/pending/repeated-byname.html).
+If you want to read all our previous minutes, head out to [the official website](http://docs.scala-lang.org/sips/minutes-list.html).
+
+Did you know that each month, we conduct these SIP meetings
+on-air? You can tune in and ask questions to the SIP committee, and have
```

Comment:
about live questions, I don't think we should allow/encourage this, actually. Community feedback should be provided in advance, it's too distracting to have it come in in the middle of a meeting. And in practice, we aren't actually doing it anyway.